index.html

```diff
@@ -24,19 +27,159 @@ <h5>CB</h5>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <button type="button" class="seeProject">See Project</button>
+        </div>
+        <!-- card 2 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-3.png" alt="tonic">
+            <h2>Multi-Post Stories</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <button type="button" class="seeProject">See Project</button>
+        </div>
+        <!-- card 3 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-4.png" alt="tonic">
+            <h2>Tonic</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <button type="button" class="seeProject">See Project</button>
+        </div>
+        <!-- card 4 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-5.png" alt="tonic">
+            <h2>Multi-Post Stories</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
```

Comment:
- [ ] As pointed by the previous reviewer you need here to use `<ul>` tag instead of a `<div>` tag, this is one of the main requirements and that will make your code more semantic. Kindly work on refactoring that🙏🏼